src/Table/Teradata/TeradataTableQueryBuilder.php

```diff
@@ -135,17 +135,27 @@ public function getCreateTableCommand(
         // TODO add settings to TABLE such as JOURNAL etc...
         return sprintf(
             'CREATE MULTISET TABLE %s.%s, FALLBACK
-(%s);',
+(%s) %s;',
             TeradataQuote::quoteSingleIdentifier($schemaName),
             TeradataQuote::quoteSingleIdentifier($tableName),
-            $columnsSql
+            $columnsSql,
+            // NoPI table support duplications in table
```

Comment:
To bych řekl, že není pravda, když tam není PK tak by to nemělo vadit, ale nic proti tomu u staging tabulek pro load csv bysme to měli určitě používat. :+1:

Reply:
hmm, nekde me to presne takto potkalo, ale jeste to pogooglim at to mame podlozene